docs/overlays/elasticsearch-shared-overlays.yaml

```diff
@@ -36,6 +36,7 @@ actions:
             Connectors are Elasticsearch integrations that bring content from third-party data sources, which can be deployed on Elastic Cloud or hosted on your own infrastructure:
             
             * Native connectors are a managed service on Elastic Cloud
+            
             * Connector clients are self-managed on your infrastructure
```

Comment:
```suggestion
            * Elastic managed connectors (Native connectors) are a managed service on Elastic Cloud
            
            * Self-managed connectors (Connector clients) are self-managed on your infrastructure
```

Reply:
🚗We're trying to simplify the taxonomy and phase out these slightly confusing original names